src/main/java/eu/dissco/backend/repository/AnnotationRepository.java

```diff
@@ -49,6 +49,14 @@ public Optional<Annotation> getActiveAnnotationForUser(String id, String userId)
         .fetchOptional(this::mapToAnnotation);
   }
 
+  public Optional<Annotation> getActiveAnnotation(String id) {
```

Comment:
I would probably have combined this with the one above and made the `.and(ANNOTATION.CREATOR.eq(userId))` in the condition